react/ActionMenu/index.jsx

```diff
@@ -166,9 +165,47 @@ ActionMenu.propTypes = {
   onClose: PropTypes.func
 }
 
+const ActionMenuHeader = ({ icon, filename, extension }) => {
+  return (
+    <Media className={styles['c-actionmenu-header']}>
+      {icon && (
+        <Icon
+          className={cx(styles['c-actionmenu-header-icon'], 'u-mr-1')}
+          icon={icon}
+          width={30}
+          height={30}
+        />
+      )}
+      <Bd className={styles['c-actionmenu-header-text']}>
+        <Bold
+          tag="span"
+          ellipsis
+          className={styles['c-actionmenu-header-filename']}
+        >
+          {filename}
+        </Bold>
+        {extension && (
+          <Bold tag="span" className="u-coolGrey">
+            {extension}
+          </Bold>
+        )}
```

Comment:
It seems it is more a FileActionMenuHeader than a generic ActionMenuHeader. Can you extract the file specific properties so that ActionMenuHeader is generic and can serve other applications than Drive ?

Reply:
I don't understand what you mean.

Reply:
I think `ActionMenuHeader` can be useful for other things than `File`.  For example in Banks, it could be useful to show the title of a transaction.

The way it is implemented now, it is very coupled to files (since it has `extension`, `filename` etc..). So for me, there are two solutions. Either rename the component to `FileActionMenuHeader` or make the `ActionMenuHeader` more generic and use it  to implement a FileActionMenuHeader than can be useful for Drive.